.golangci.yml

```diff
@@ -20,7 +20,8 @@ linters:
 linters-settings:
   exhaustive:
     default-signifies-exhaustive: true
-
+  staticcheck:
+    checks: ["all", "-SA1019"]
```

Comment:
What's this? Did you include this on purpose? If so, might be useful to make this a separate commit and explain why this is added.

Reply:
Ah yes. See https://github.com/jesseduffield/lazygit/pull/3040/commits/3e2ef84d566678ffaa6afa120c6881b558eb2227

Reply:
Nice, thanks.

Reply:
Ah yes. See https://github.com/jesseduffield/lazygit/pull/3040/commits/3e2ef84d566678ffaa6afa120c6881b558eb2227